### ISSUE TITLE
The cursor is always on the left?please!

### ISSUE THREAD

**Issue by dahvlhldle** [memspace/zefyr#61]
## Steps to Reproduce

1. ...
2. ...
3. ...

## Logs

```
```

```
```

**Comment by pulyaevskiy** [comment]
I assume this is about RTL support. Yes, it's not supported yet, sorry. Will try to address this in one of the upcoming releases.

**Comment by gilgold** [comment]
RTL Support would be great, thanks!

**Comment by pulyaevskiy** [comment]
I might need some help to identify the exact issue here as I have practically 0 experience in RTL. 

I've just tried to set Directionality to RTL and see how it looks like comparing to regular Flutter text fields and aside from some minor rendering bugs it seems to be behaving very much the same. 

Anything in particular I'm missing? Here is a recording of my comparison between standard Flutter text field and Zefyr:
![kapture 2019-01-02 at 12 30 54](https://user-images.githubusercontent.com/441756/50611193-6720f680-0e8a-11e9-8d02-ce99c3b8f044.gif)

**Comment by gilgold** [comment]
Sorry for the delay in response.
When talking about RTL support, it is not simply setting the Directionality.
Most editors (for example, MS Word) have the option to align a paragraph to the right or left.
This is not a simple "text align" feature, rather it changes the directionality for that specific paragraph. The most notable way to see the difference between text align and paragraph align is when applying bullet points.
With paragraph align = right: the bullets appear on the right side of the text lines
With text align = right: the text is aligned to the right but the bullets appear at the left side.

So basically what we need is a way to align a paragraph to the the right / left within the document.
I believe quilt.js has that option built in.

**Comment by pulyaevskiy** [comment]
I see, thanks for the details.

So for RTL bullet lists or other lists (like quotes) are supposed to have their visual elements drawn on the right. This makes sense.

Note that zefyr does not support text align for text (e.g. left, right, center or justify) so alignment is always the same: for LTR it's left, for RTL it's right. Which I assume is correct?

**Comment by gilgold** [comment]
Yes that is correct, would be great if it did support text alignments / paragraph alignment (allot of time people who write in RTL languages need to include content that is LTR for example code or English words)

**Comment by pulyaevskiy** [comment]
> would be great if it did support text alignments / paragraph alignment

Good points, thanks! This is definitely something I'd like zefyr to support at some point.

**Comment by osaxma** [comment]
Will this be supported by any chance in the new Zefyr?

**Comment by mehrdad-shokri** [comment]
@pulyaevskiy another point is that ordered/unordered lists should follow it's text Direction. Right now it rendered on the left

**Comment by pulyaevskiy** [comment]
I'll close this as duplicate of #475 . Zefyr `1.0.0-rc.1` provides better RTL support thanks to contributions from @Amir-P but there still seems to be things which need to be fixed. I'd encourage to leave a comment in #475 with everything that you believe is still missing.
